pages/linux/eww.md

```diff
@@ -0,0 +1,28 @@
+# eww
+
+> Implement your own custom widgets.
```

Comment:
That kind of widget is meant here? I assume these are widgets for a graphical desktop environment? Is it for X? Or Wayland? Or both. I think it would be good to make this description a bit longer.

Reply:
I changed it to something similar in the GitHub. Lmk if it should be even more explicit.